configs/components/jsbach/jsbach.yaml

```diff
@@ -484,8 +484,8 @@ outdata_files:
         "[[streams-->STREAM]]_codes": STREAM_codes
 
 outdata_sources:
-        "[[streams-->STREAM]]": ${general.expid}_${start_date!syear!smonth}.${start_date!sday}_STREAM
-        "[[streams-->STREAM]]_codes": ${general.expid}_${start_date!syear!smonth}.${start_date!sday}_STREAM.codes
+        "[[streams-->STREAM]]": ${general.expid}_${start_date!syear}*.${start_date!sday}_STREAM
+        "[[streams-->STREAM]]_codes": ${general.expid}_${start_date!syear}*.${start_date!sday}_STREAM.codes
```

Comment:
I'm curious about this change @denizural, I'm totally fine with it, but can you elaborate on why it was necessary?